Makefile

```diff
@@ -187,14 +187,15 @@ update-bindata: go-bindata ## Update bindata.go file.
 	rm -rf ./$(GATEKEEPER_MANIFEST_DIR)-rendered
 	$(MAKE) fmt
 
-GATEKEEPER_IMAGE ?= quay.io/gatekeeper/gatekeeper
+GATEKEEPER_IMG_BASE ?= quay.io/gatekeeper/gatekeeper
+GATEKEEPER_IMG ?= $(GATEKEEPER_IMAGE_BASE):v$(GATEKEEPER_VERSION)
 
 .PHONY: update-gatekeeper-image
 update-gatekeeper-image: ## Update Gatekeeper image in manifests.
-	yq 'select(.kind == "Deployment") \
+	yq "select(.kind == \"Deployment\") \
```

Comment:
We may have to write a Python script instead if `yq` isn't available in the UBI image.